packages/server/lib/utils/tests.ts

```diff
@@ -176,3 +182,20 @@ export async function getConnectSessionToken(api: Awaited<ReturnType<typeof runS
     if (!token) throw new Error('No connect session token');
     return token;
 }
+
+export async function authenticateUser(api: Awaited<ReturnType<typeof runServer>>, user: DBUser, password: string = 'Password123!'): Promise<string> {
+    const { res } = await api.fetch('/api/v1/account/signin', {
+        method: 'POST',
+        body: { email: user.email, password }
+    });
```

Comment:
I guess the cookie is mandatory and the easiest way to do it, we can't emulate that by creating our own session to avoid a round-trip?